### PR TITLE
Prepare Attestation SDK for May Release.

### PR DESCRIPTION
**Pull request by LarryOsterman**
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [X] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [X] Doxygen docs
- [X] Unit tests
- [X] No unwanted commits/changes
- [X] Descriptive title/description
  - [X] PR is single purpose
  - [] Related issue listed
- [X] Comments in source
- [X] No typos
- [X] Update changelog
- [X] Not work-in-progress
- [X] External references or docs updated
- [X] Self review of PR done
- [X] Any breaking changes?

### CODE REVIEW

File: sdk/attestation/azure-security-attestation/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2022-05-10)
 
 ### Features Added
```
Comment (ahsonkhan) [inline sdk/attestation/azure-security-attestation/CHANGELOG.md:5]: nit: I think we can remove this heading all together given we have no new features, and reduce the noise-to-signal ratio for customers reading the changelog. The single bullet here isn't really relevant to users.

Comment (ahsonkhan) [reply sdk/attestation/azure-security-attestation/CHANGELOG.md:5]: Also, this text seems redundant and can be removed, along with surrounding white space (in the breaking change section):
> Breaking Changes from API Review

**Review by ahsonkhan — APPROVED** [review]

File: sdk/attestation/azure-security-attestation/CHANGELOG.md (hunk 1 above)
Comment (LarryOsterman) [reply sdk/attestation/azure-security-attestation/CHANGELOG.md:5]: Maybe? The scripts get really annoyed if the various elements aren't all there. I'm concerned that removing the Features Added section is going to upset the release scripts.

And I wanted to express that all the breaking changes in this list are a consequence of API Review and not just me being completely random in my changes.

Comment (ahsonkhan) [reply sdk/attestation/azure-security-attestation/CHANGELOG.md:5]: > Maybe? The scripts get really annoyed if the various elements aren't all there. I'm concerned that removing the Features Added section is going to upset the release scripts.


The release scripts shouldn't care and be resilient to such headings not being there. We have done them in the past, without issues. For example, see:
https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/core/azure-core/CHANGELOG.md#131-2021-11-05

> And I wanted to express that all the breaking changes in this list are a consequence of API Review and not just me being completely random in my changes.

The intended audience for the changelog are our customers. That detail shouldn't be relevant to them :)

**Review by ahsonkhan — APPROVED** [review]

**Pull request merged**